packages/browser/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -86,48 +86,45 @@ export default class AuthorizationCodeWithPkceOidcHandler
     const { redirector } = this;
     const storage = this.storageUtility;
 
-    await oidcClientLibrary.createSigninRequest().then((req: SigninRequest) => {
-      return (
-        Promise.all([
-          // We use the OAuth 'state' value (which should be crypto-random) as
-          // the key in our storage to store our actual SessionID. We do this
-          // 'cos we'll need to lookup our session information again when the
-          // browser is redirected back to us (i.e. the OAuth client
-          // application) from the Authorization Server.
-          // We don't want to use our session ID as the OAuth 'state' value, as
-          // that session ID can be any developer-specified value, and therefore
-          // may not be appropriate (since the OAuth 'state' value should really
-          // be an unguessable crypto-random value).
+    try {
```

Comment:
To verify: this change just removes an unused import and rewrites this to async/await syntax - there's no behavioural change I need to look out for, right?

Reply:
Absolutely, this used a mix of `await` and `.then`, which can sometimes result in confusing code, so at least now things are clearer. I initially suspected such a confusion caused by this could be causing the issue, and that wasn't the case, but I still think it's cleaner now :).